### ISSUE TITLE
keepass@2.49: %ADDDATA%\KeePass\KeePass.config.xml is left after uninstall

### ISSUE THREAD

**Issue by KKunai** [lukesampson/scoop-extras#6848]
`%ADDDATA%\KeePass\KeePass.config.xml` is left after running `scoop uninstall keepass`.  
I think this file may cause confusion like the configurations persists even if I do `scoop install keepass` after `scoop uninstall keepass -p` and delete `$persist_dir\\KeePass.config.xml`.

`%ADDDATA%\KeePass\KeePass.config.xml` has per-user configurations, so I think this should be deleted while uninstall.

https://keepass.info/help/base/configuration.html

**Comment by KKunai** [comment]
manifest: https://github.com/lukesampson/scoop-extras/blob/master/bucket/keepass.json

**Comment by HUMORCE** [comment]
https://github.com/lukesampson/scoop-extras/blob/91c1749d100f4fc5e772821ec80db645f1000c50/bucket/keepass.json#L8-L11

scoop did not copy or move configfile to other directories, just stores between `$dir` and `$persist_dir`.

**Comment by KKunai** [comment]
Yes, scoop does not copy or move config file to `%APPDATA%\KeePass\KeePass.config.xml`, but KeePass.exe creates `%APPDATA%\KeePass\KeePass.config.xml`.

**Comment by HUMORCE** [comment]
![screenshot_20210914144505](https://user-images.githubusercontent.com/4353480/133279631-82ba3b33-e077-451d-b042-94ed45a46022.png)

![screenshot_20210914145241](https://user-images.githubusercontent.com/4353480/133280991-faa4eadc-4f08-47e2-8b50-978479b6b29f.png)

not really.

**Comment by KKunai** [comment]
Thank you for your confirmation.
It's may be caused by my bloken permission of `$dir\\KeePass.config.xml`.
According to this discussion, if installation directory is not writable, KeePass writes config to `%APPDATA%\KeePass\KeePass.config.xml`.
https://sourceforge.net/p/keepass/discussion/329220/thread/e738b9ded1/#3e92
![image](https://user-images.githubusercontent.com/68718305/133531470-8119bd52-a917-4cc2-b729-381943e8f206.png)

I have set write permission to my user, and I could finally use `$dir\\KeePass.config.xml`.
